src/Network/NWConnection.cs

```diff
@@ -530,7 +530,13 @@ public void SendIdempotent (byte [] buffer, NWContentContext context, bool isCom
 		}
 
 		[DllImport (Constants.NetworkLibrary)]
-		extern static string nw_connection_copy_description (IntPtr handle);
+		extern static IntPtr nw_connection_copy_description_ptr (IntPtr handle);
```

Comment:
This won't work: the code will now try to find the function `nw_connection_copy_description_ptr` in native code, which doesn't exist. You need to specify the EntryPoint in the DllImport attribute:

```cs
[DllImport (Constants.NetworkLibrary, EntryPoint = "nw_connection_copy_description")]
```